ngraph/frontend/frontend_manager/src/frontend_manager.cpp

```diff
@@ -32,7 +25,7 @@ class FrontEndManager::Impl
 
     FrontEnd::Ptr loadByFramework(const std::string& framework, FrontEndCapFlags fec)
     {
-        FRONT_END_ASSERT(m_factories.count(framework))
+        FRONT_END_INITIALIZATION_CHECK(m_factories.count(framework));
```

Comment:
Suggest to add error message, "frontend for framework '<name>' is not found"

Reply:
done